src/blocks/product-card.css

```diff
@@ -0,0 +1,113 @@
+/* #region Product Card */
+.product-card {
+  display: flex;
+  flex-direction: column;
+
+  box-sizing: border-box;
+  padding: 0 16px;
+  width: 200px;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product-card__image {
+  align-self: center;
+  height: 134px;
+  width: 160px;
+  margin: 32px 0;
+}
+
+.product-card__name {
+  margin: 8px 0;
+}
+
+.product-card__device-model {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: var(--main-accent);
+}
+
+.product-card__code {
+  margin-top: 4px;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.product-card__rating {
+  display: flex;
+  justify-content: space-between;
+  margin: 8px 0;
+}
+
+.product-card__reviews {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+
+  display: flex;
+  align-items: flex-end;
+  text-align: right;
+
+  color: var(--main-accent);
+}
+
+.product-card__price-section {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  gap: 60px;
```

Comment:
1 - this gap don't make any sense since you've used `space-beetween`
2 - better to avoid `gap`, it's not supported by some versions of browser (also browser in tests is not supporting it)